modules/browser/BrowserCapabilities.js

```diff
@@ -270,7 +279,15 @@ export default class BrowserCapabilities extends BrowserDetection {
      * @returns {boolean}
      */
     usesAdapter() {
-        return this.usesNewGumFlow();
+        return !this.isFirefox() && !this.isReactNative();
+    }
+
+    /**
+     * Checks if the browser uses RIDs/MIDs for siganling the simulcast streams
+     * to the bridge instead of the ssrcs.
+     */
+    usesRidsForSimulcast() {
```

Comment:
Is this in preparation for part 2?

Reply:
I need this here  - https://github.com/jitsi/lib-jitsi-meet/pull/1062/files#diff-902544c7807a8d275d7f45764155d76bR322
so that we don't create ssrc-groups for rids in the jingle message.